pkg/code-generator/schemagen/crd.go

```diff
@@ -62,12 +56,12 @@ func (c *CrdWriter) ApplyValidationSchemaToCRD(crd apiextv1beta1.CustomResourceD
 	})
 }
 
-func getFilenameForCRD(crd apiextv1beta1.CustomResourceDefinition) string {
-	return fmt.Sprintf("%s_%s_%s.yaml", crd.Spec.Group, crd.Spec.Version, crd.Spec.Names.Kind)
+func getFilenameForCRD(crd apiextv1.CustomResourceDefinition) string {
+	return fmt.Sprintf("%s_%s_%s.yaml", crd.Spec.Group, crd.Spec.Versions[0].Name, crd.Spec.Names.Kind)
```

Comment:
now that we support (or will support) multple versions of the CRD. Does it make sense to make this file name just be ${GROUP}_${KIND}?

Reply:
Since we decided to not support multiple versions yet, I think we should keep the name of the version we used in the filename.